pkg/apps/blackduck/latest/containers/configmap.go

```diff
@@ -34,12 +35,41 @@ func (c *Creater) GetConfigmaps() []*components.ConfigMap {
 
 	var configMaps []*components.ConfigMap
 
-	hubConfig := components.NewConfigMap(horizonapi.ConfigMapConfig{Namespace: c.hubSpec.Namespace, Name: "blackduck-config"})
+	hubConfig := components.NewConfigMap(horizonapi.ConfigMapConfig{Namespace: c.hubSpec.Namespace, Name: util.GetResourceName(c.name, "blackduck-config", c.config.IsClusterScoped)})
+
 	hubData := map[string]string{
 		"RUN_SECRETS_DIR": "/tmp/secrets",
 		"HUB_VERSION":     c.hubSpec.Version,
 	}
 
+	if !c.config.IsClusterScoped {
```

Comment:
Actually it's not required if it's cluster scope. Trying to minimize the merging of map..

Reply:
The amount env variables being merged isn't that large.  I can see consistency in code and deployment to be valuable here if we won't break ourselves.  Maybe log an issue to look at removing this if condition and do extensive testing with cluster and ns scope and these env variables.  My one concern is that we end up setting these env in cluster scope and something breaks because a name is different.